modules/aws_vm/main.tf

```diff
@@ -81,6 +81,7 @@ resource "aws_instance" "vm" {
     delete_on_termination = var.os_disk_delete_on_termination
     iops                  = var.os_disk_iops
     encrypted             = var.enable_ebs_encryption
+    tags                  = merge(var.tags, tomap({ Name : "${var.name}-root-vol" }))
```

Comment:
Would suggest changing the format to match the other additions for consistency.

Reply:
@thpang Fixed the format of the root block device tags so that it matches the rest of the changes